lib/matplotlib/font_manager.py

```diff
@@ -1276,14 +1276,14 @@ def _findfont_cached(self, prop, fontext, directory, fallback_to_default,
                 # This is a hard fail -- we can't find anything reasonable,
                 # so just return the DejuVuSans.ttf
                 warnings.warn(
-                    'findfont: Could not match %s. Returning %s' %
+                    'findfont: Could not match %s. Returning %s.' %
                     (prop, self.defaultFont[fontext]),
                     UserWarning)
                 result = self.defaultFont[fontext]
         else:
             _log.debug(
-                'findfont: Matching %s to %s (%s) with score of %f' %
-                (prop, best_font.name, repr(best_font.fname), best_score))
+                'findfont: Matching %s to %s (%r) with score of %f.' %
```

Comment:
Couldn't you replace the manual `%` and replace with `,` (and pull the items out of the tuple) and let the logger handle the string interpolation?

Reply:
fixed